staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go

```diff
@@ -210,7 +210,7 @@ var (
 		flowcontrol.PriorityLevelConfigurationSpec{
 			Type: flowcontrol.PriorityLevelEnablementLimited,
 			Limited: &flowcontrol.LimitedPriorityLevelConfiguration{
-				AssuredConcurrencyShares: 20,
+				AssuredConcurrencyShares: 100,
```

Comment:
The believe is that we have many more of these than we do workload-high clients?

Reply:
@deads2k `global-default` matches workloads that are running as non SA accounts. If I am not mistaken they are fewer compared to `service-accounts` which is basically all workloads (minus `workload-high`) running inside the cluster with service accounts.

`service-accounts` workload matches the `workload-low` priority level.
The concurrency pool of `workload-low` is `20` whereas that of `global-default` is `100`, This PR swaps the values so `service-accounts` workloads have more concurrency share.

As far as `workload-high` is concerned, I have not seen `workload-high` workloads starve in my testing where we had about 13K Pods running at any time and a high namespace and pod churn rate. It was a 250-node cluster. 
At the same time,  I recently came across a CI job where `workload-high` was starving because a certain admission controller `OwnerReferencesPermissionEnforcement` was taking order of seconds to respond and triggering p&f to throttle `workload-high`.